core/services/pipeline/graph.go

```diff
@@ -70,11 +71,6 @@ func (g TaskDAG) TasksInDependencyOrder() ([]Task, error) {
 			return nil, err
 		}
 
-		err = task.SetDefaults(node.attrs, g, *node)
```

Comment:
What did this actually do and why is it no longer necessary>

Reply:
Because we're currently calculating task parameters at the time that the task run is executed, `SetDefaults` (which was a special case for `median`, a task that needed to handle the case where its `allowedFaults` param was unset, and needed to default to something calculated dynamically) is no longer necessary